qemu/tests/cfg/uefishell.cfg

```diff
@@ -18,6 +18,29 @@
             command_connect = "connect -r"
             command_oprom = "dh -d -v -p SimpleNetwork"
             check_result_oprom = "Controller Name.*:.*iPXE"
+        - check_network:
+            bootindex_nic1 = 1
+            time_interval = 3
+            variants:
+                - with_ipv4:
+                    test_scenarios = "connect config show ping"
+                    command_connect = "connect -r"
+                    command_config = "ifconfig -s eth0 dhcp"
+                    command_show = "ifconfig -l eth0 dhcp status"
+                    command_ping = "ping "
+                    command_ping_args = "get_host_ip_ver4()"
+                    check_result_show = "ipv4.*address.*:.*(\d+.\d+.\d+.\d+)"
+                    check_result_ping = "0.*packet loss"
+                - with_ipv6:
+                    test_scenarios = "connect config6 show6 ping6"
+                    command_connect = "connect -r"
+                    command_config6 = "ifconfig6 -s eth0 auto"
+                    command_show6 = "ifconfig6 -l eth0"
+                    command_show6_output = ipv6_addr
+                    command_ping6 = "ping6 -s "
+                    command_ping6_args = "get_host_ip_ver6()"
```

Comment:
command_ping6_args = "form_ping6_args"

def form_ping6_args():
    out = uefishell_test.send_command('ifconfig6 -l eth0')
    get src_ipv6 from out
    target_ipv6 = utils_net.get_host_ip_address('ipv6')
    return "src_ipv6 target_ipv6" string

So command_show6_output can be removed, for its value 'ipv6_addr' is nothing but a flag

Reply:
agree with you, this way is better. updated, thanks.